revised to create TDM and a dataframe for text analysis.

alspur opened the pull request:
Took a stab at addressing the issues you noted in your blog - chapter 3 from the book "Machine Learning for Hackers" was very helpful in figuring this out!

alspur committed:
revised to create TDM and a dataframe for text analysis.

ulfelder commented:
Thanks, Alex! I'm actually interested in tracking change over time, so I
want to keep the reports as separate columns, but your trick for creating
the tdm worked great. I've now got a script that uses that and then makes a
data frame from the results so I can do time-series plots or heat maps and
such. -JU

On Wed, Feb 11, 2015 at 11:39 AM, Alex S. notifications@github.com wrote:

> Took a stab at addressing the issues you noted in your blog - chapter 3
> from the book "Machine Learning for Hackers" was very helpful in figuring
> 
> ## this out!
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/ulfelder/national-security-strategy/pull/1
> Commit Summary
> - revised to create TDM and a dataframe for text analysis.
> 
> File Changes
> - _M_ nss.explore.R
>   https://github.com/ulfelder/national-security-strategy/pull/1/files#diff-0
>   (47)
> 
> Patch Links:
> - https://github.com/ulfelder/national-security-strategy/pull/1.patch
> - https://github.com/ulfelder/national-security-strategy/pull/1.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ulfelder/national-security-strategy/pull/1.

alspur commented:
Glad I could help and best of luck with the rest of your analysis - I look forward to seeing the final product!

Cheers,

Alex

—
AMS

> On Feb 11, 2015, at 4:10 PM, Jay Ulfelder notifications@github.com wrote:
> 
> Thanks, Alex! I'm actually interested in tracking change over time, so I
> want to keep the reports as separate columns, but your trick for creating
> the tdm worked great. I've now got a script that uses that and then makes a
> data frame from the results so I can do time-series plots or heat maps and
> such. -JU
> 
> On Wed, Feb 11, 2015 at 11:39 AM, Alex S. notifications@github.com wrote:
> 
> > Took a stab at addressing the issues you noted in your blog - chapter 3
> > from the book "Machine Learning for Hackers" was very helpful in figuring
> > 
> > ## this out!
> > 
> > You can view, comment on, or merge this pull request online at:
> > 
> > https://github.com/ulfelder/national-security-strategy/pull/1
> > Commit Summary
> > - revised to create TDM and a dataframe for text analysis.
> > 
> > File Changes
> > - _M_ nss.explore.R
> >   https://github.com/ulfelder/national-security-strategy/pull/1/files#diff-0
> >   (47)
> > 
> > Patch Links:
> > - https://github.com/ulfelder/national-security-strategy/pull/1.patch
> > - https://github.com/ulfelder/national-security-strategy/pull/1.diff
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/ulfelder/national-security-strategy/pull/1.
> > 
> > —
> > Reply to this email directly or view it on GitHub https://github.com/ulfelder/national-security-strategy/pull/1#issuecomment-73965465.
